keel-core/src/test/kotlin/com/netflix/spinnaker/keel/services/ApplicationServiceTests.kt

```diff
@@ -1,16 +1,10 @@
-package com.netflix.spinnaker.keel
+package com.netflix.spinnaker.keel.services
```

Comment:
I removed all dependencies from `ApplicationServiceTests` with the `keel-ec2-plugin` module, which allowed me to move it, and the service class itself, to `keel-core`.